emukit/core/parameter_space.py

```diff
@@ -147,3 +147,17 @@ def check_points_in_domain(self, x: np.ndarray) -> np.ndarray:
             in_domain = np.all([in_domain, param_in_domain], axis=0)
             encoding_index += param.dimension
         return in_domain
+
+    def sample_uniform(self, point_count: int) -> np.ndarray:
```

Comment:
This is a brilliant way to resolve our problems with dependency on Random design everywhere. Thanks!

Reply:
Thank you!